test/dependency/CMakeLists.txt

```
@@ -1,19 +1,19 @@
-cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.14...3.20)
```

Not sure I like specifying a max version as it set's a mix of policies that are not easy to track.

```suggestion
cmake_minimum_required(VERSION 3.14 FATAL_ERROR) 
```

---

No,  minimum is cmake3.14!
There is no maximum, current is 3.20, and I want the newest available policiy setting between 3.14 and 3.20.
I have tested with 3.19, so I am sure, it works the next view cmake releases.

---

This is a pointless change. If anything, that `FATAL_ERROR` should be removed, as it has been [ignored since 2.6](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) and [versions older than 2.8.12 are deprecated](https://cmake.org/cmake/help/latest/release/3.19.html#deprecated-and-removed-features) anyway.

---

Fair point. What is the advantage of specifying a max version additionally to a min version, though? As far as I understand it sets some policy changes, but this may lead to different behaviors on versions lower than max right?

---

Its use is when you have code that is conditional on policies, but this project doesn't have anything of sorts.